spring-integration-hazelcast/.gitignore

```diff
@@ -0,0 +1,4 @@
+/target/
+/.settings/
+/.classpath
+/.project
```

Comment:
Hi @artembilan,

Many thanks for all your feedback and time. I really appreciated.

I have just started to work on them ;)